src/header.tsx

```diff
@@ -144,6 +146,7 @@ export const HeaderMenu: React.FC<HeaderMenuProps> = ({ close, items, label, bre
 export type HeaderMenuItemProps = JSX.IntrinsicElements["li"] & {
   icon?: JSX.Element;
   wrapper?: JSX.Element;
+  ref?: RefObject<HTMLLIElement | HTMLDivElement>
```

Comment:
Ehm sorry, only noticing this now. This can be removed right? The `ref` is passed in a special way, using `forwardRef` below?

Reply:
Forgot to comment on this but it's been on my mind. I'm not exactly sure why, but removing this results in an error  when using the component. I think it's because of the way `ref.current` can be assigned different types in the component 🤨

Reply:
I tried around and it seems you can do this instead:

```ts
export type HeaderMenuItemProps = Omit<JSX.IntrinsicElements["li"], "ref"> & {
  icon?: JSX.Element;
  wrapper?: JSX.Element;
};
```

If I triggered the same error you are talking about, it was apparently caused by this.